docs/community/devsetup.md

```diff
@@ -1,121 +0,0 @@
-# Development Setup
```

Comment:
CONTRIBUTING.md points to this: https://github.com/awslabs/gluonts/blob/dev/CONTRIBUTING.md

maybe we leave this here and update it?

Reply:
Do you mean also remove it from there?

Reply:
Either that, or we just keep the page around and update it with current instructions